packages/compass-preferences-model/src/feature-flags.ts

```diff
@@ -81,7 +81,7 @@ export const featureFlags: Required<{
    * Epic: COMPASS-7584
    */
   showGenAIPassSampleDocumentsSetting: {
-    stage: 'development',
+    stage: 'released',
     description: {
       short: 'Enable showing the sample document gen ai setting.',
```

Comment:
```suggestion
      short: 'Enable showing the sample document Generative AI setting.',
```

Should this even be a feature flag? To me this makes sense as just a regular setting that we could expose

Reply:
There is a setting that this feature flag, when enabled, shows. So this feature flag is for showing the setting 😅 
<img width="888" alt="Screenshot 2024-06-03 at 1 48 39 PM" src="https://github.com/mongodb-js/compass/assets/1791149/c3ae55e4-d037-4c68-acfa-0c69f977b563">

Reply:
We could probably drop the FF entirely in this case? or we think we would roll this one back?

Reply:
I mean, we can keep it, just feels a bit like we don't need two separate settings in a situation like this 🙂

Reply:
We went for a feature flag so that product folks could try out the setting before we released it. In general I agree, a bit of a specialized case here. I think dropping the feature flag makes sense, I don't see it being rolled back. I'll push up that change momentarily.